apps/hannk/delegate/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+# --------------- registrar (static-library) delegate
+
+add_library(hannk_delegate_registrar STATIC
+            hannk_delegate_provider.cpp
+            hannk_delegate.cpp)
+target_link_libraries(hannk_delegate_registrar PRIVATE
+                      interpreter
+                      error_util
+                      hannk_log_tflite
+                      tensorflowlite_headers)
+target_include_directories(hannk_delegate_registrar PRIVATE
+                           $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}>
```

Comment:
Do you really mean `CMAKE_SOURCE_DIR`, here? Or do you mean `CMAKE_CURRENT_SOURCE_DIR` or `hannk_SOURCE_DIR`?